backend/tests/integration/migrator/test_migrator_cli.py

```diff
@@ -220,3 +220,15 @@ def test_migrate_continuously(mock_migration, mock_handler, inspire_app, cli):
     assert result.exit_code == 0
     assert mock_handler.call_count == 3
     assert mock_migration.call_count == 2
+
+
+def test_regression_migration_step_exits_without_error_when_steps_ends(
+    inspire_app, cli
+):
+    create_record("lit")
+    result = cli.invoke(["migrate", "mirror_step", "-s", "2"])
+    assert 0 == result.exit_code
+    assert (
+        "stop_after_one_step() takes 0 positional arguments but 1 was given"
+        not in result.output
+    )
```

Comment:
IMO just ``assert 0 == result.exit_code`` is enough, WDYT?

Reply:
:+1: